CHANGELOG.md

```diff
@@ -9,9 +9,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ### Added -->
 
-<!-- ### Changed -->
+### Changed
+
+- When adding subprojects, projected budgets are not mandatory anymore [#229](https://github.com/openkfw/TruBudget/issues/229)
+
+
+### Fixed
+
+- Configuration for excel export in docker-compose YAML file [#223](https://github.com/openkfw/TruBudget/issues/223)
```

Comment:
Please go for either one of the following.

Either this:

```suggestion
- configuration for excel export in docker-compose YAML file. [#223](https://github.com/openkfw/TruBudget/issues/223)
```

or, even better, this:

```suggestion
- Instead of building, use the existing `trubudget/excel-export` image in "master" docker-compose file. [#223](https://github.com/openkfw/TruBudget/issues/223)
```